File: 2.1SEM/Python/50sual FM (Olimpiada)/A31.py
n = int(input("eded: "))
temp = n
while temp>=1:
    if( n % (temp%10) !=0 ):
        print("no")
        break
    temp=temp//10
else: print("yes")

#4reqemleri ededin oz reqemlerine bolunub bolunmediyini tapin

File: 2.1SEM/C/50sual FM (Olimpiada)/A31.cpp
#include<stdio.h>
int main(){
int n,temp;
bool test= true;
scanf("%d",&n);
temp = n;
while(temp>0){
	if(n%(temp%10) !=0){
		test = false;
		break;
	}
	temp/=10;
}
if(test) printf("yes");
else printf("no");
return 0;
}

//4reqemleri ededin oz reqemlerine bolunub bolunmediyini tapin
